local-meetups/framework.md

```diff
@@ -0,0 +1,72 @@
+2. [Problem statement](#problem-statement)
+3. [Introducing Local OSPO Meetups powered by TODO Group](#introducing-local-ospo-meetups-powered-by-todo-group)
+   + [The Framework](#the-framework)
+   + [Meetup expectations](#meetup-expectations)
+   + [Local Organizer Requirements and Responsibilities](#local-organizer-requirements-and-responsibilities)
+5. [Interested?](#interested)
+
+
+
+## About OSPOlogy 
+OSPOlogy is an initiative under the TODO group that offers best practice-sharing online workshops that are open to all, from folks starting their OSPOs to seasoned OPSOers! These sessions are typically led by a TODO member covering one topic or sharing their organization’s OSPO journey.
+For further details see https://github.com/todogroup/ospology.
+
+## Problem statement
+TODO Group members requested local events to find and network with OSPOers near their area, to different OSPO topics.
+Introducing OSPOlogy Meetups, powered by TODO Group.
+
+We are introducing a new framework for in-person local meetups. OSPO Local Meetups aim to provide a forum for like-minded professionals to share knowledge and experience on OSPOs and Open Source Management within specific regions.
+
+## Introducing Local OSPO Meetups powered by TODO Group
+
+We are introducing a new framework for in-person local meetups. TODO Local Meetups aim to provide a forum for like-minded professionals to share knowledge and experience on OSPOs and Open Source Management within specific regions.
+
+### The Framework
+
+In a nutshell, the framework defines the following local meetup requirements:
+
+* Hybrid gatherings that are open to TODO members and the public
+* Hosted by a Local TODO Organizer
+* Follow [Chatham House Rules](https://www.chathamhouse.org/about-us/chatham-house-rule)
+  * Recordings are not allowed
+  * The information received can be shared as meet-up notes, but does not reveal the identity of who said it
+* Provides a forum for like-minded professionals to share knowledge and experience on OSPOs and Open Source Management
+* Encourages vendor-neutraly
+* Encourages diversity of attendees and regions to participate including hosting meetups in local languages
+* TODO community participants can propose a new local TODO meetup
+* Needs to have a Local TODO Organizer
+* Is under Linux Foundation [policies](https://www.linuxfoundation.org/policies/) and [code of conduct](https://events.linuxfoundation.org/about/code-of-conduct/)
+* Participants must register for TODO Local meet-ups via [Linux Foundation’s community platform](https://community.linuxfoundation.org/todo-group-europe/).
+
+### Meetup Expectations
+
+OSPO Local meetups powered by the TODO Group is committed to vendor neutrality and aims to become a forum for like-minded professionals to share knowledge and experiences on OSPOs. Even though we understand that during a conversation, OSPO tooling and/or services examples might arise, **vendor pitches must not be part of the meetup core discussions**.
```

Comment:
Let's add a bit to this. Maybe something like what I proposed in Slack to add to or replace the part currently in bold:
Vendors may not propose solutions or use the meetups under the TODO Group umbrella for any discussions designed to generate sales for a vendor. Meetups are not intended to be vendor pitches and can’t be used for that purpose if you would like to have the meetup under the TODO Group branding.